aea/protocols/generator.py

```diff
@@ -241,16 +251,23 @@ def _message_class_str(self) -> str:
         cls_str += '        assert self.is_set("target"), "target is not set."\n'
         cls_str += '        return cast(int, self.get("target"))\n\n'
         cls_str += "    @property\n"
-        cls_str += "    def performative(self) -> str:\n"
+        cls_str += "    def performative_raw(self) -> str:\n"
```

Comment:
We should not have this property, it is unnecessary as the str version of the performative can be recovered via `str(self.performative)`

Reply:
Done